packages/react-scripts/template/.travis.yml

```diff
@@ -5,8 +5,14 @@ before_install:
   - echo -e "machine github.com\n  login $CI_USER_TOKEN" > ~/.netrc
   - echo "//code.lds.org/artifactory/api/npm/npm-fhd/:_authToken=${NPM_TOKEN}" >> ~/.npmrc
   - echo "@fs:registry=https://code.lds.org/artifactory/api/npm/npm-fhd/" >> ~/.npmrc
-  - npm i -g npm@6.8.0
-script: npm run lint && npm run test
+  - npm i -g npm@latest
```

Comment:
could we just remove this line completely now? Do we still see the problem if we just use the default version given to us by node? It sure would be nice to speed up our builds by however long it takes to npm install npm

Reply:
no, it still fails. There is no node version released that defaults to npm 6.10 yet
We could pin to node 10.15 instead, if we want to be faster.